eogrow/utils/map.py

```diff
@@ -7,6 +7,11 @@
 import subprocess
 from tempfile import NamedTemporaryFile
 from typing import Iterable, Literal, Optional
+from pathlib import Path
```

Comment:
`isort` complains about the order. Please run `pre-commit run --all-files` to fix.

Reply:
resolved

Reply:
The pre-commit run --all-files command ran successfully on my machine. Rebased the branch and pushed again. Please let me know if this looks okay now.